src/MorphoTokenEthereum.sol

```diff
@@ -16,6 +16,14 @@ contract MorphoTokenEthereum is Token {
     /// @dev The symbol of the token.
     string internal constant SYMBOL = "MORPHO";
 
+    /* EVENTS */
+
+    /// @dev Emitted whenever tokens are minted for an account.
+    event Mint(address indexed account, uint256 amount);
+
+    /// @dev Emitted whenever tokens are burned from an account.
+    event Burn(address indexed account, uint256 amount);
+
```

Comment:
Shouldn't these be in Token.sol? They are also used in Optimism

Reply:
done in #45